x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials.tsx

```diff
@@ -4,12 +4,140 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiCopy,
+  EuiButtonIcon,
+  EuiSpacer,
+  EuiButton,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import {
+  CredentialsLogic,
+  ICredentialsLogicActions,
+  ICredentialsLogicValues,
+} from './credentials_logic';
```

Comment:
See above - awesome right?
```suggestion
import { CredentialsLogic } from './credentials_logic';
```

Reply:
2c9a99a250f9766b97948fb23d40d200a427bf64